label_studio/ml_models/migrations/0010_modelinterface_skill_name.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.25 on 2024-08-13 22:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Can we avoid migrations if possible?

Reply:
This is needed, we kept the new fields added to a minimum but we do have to have a way to distinguish between types of projects at least. @pakelley can chime in w other design considerations but we discussed this as a team